src/renderer/components/ContextMenu/useContextMenu.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext, useMemo } from 'react';
+import { MenuContext } from './ContextMenu';
+import { ContextData } from './types';
+
+export default function useContextMenu(data: ContextData) {
+  const handlerCreator = useContext(MenuContext);
```

Comment:
If we make the context have the default value of `undefined` we can then do this.

```suggestion
  const handlerCreator = useContext(MenuContext);

  if (!handlerCreator) {
    throw new Error(
      'useContextMenu can only be used within MenuContext.Provider'
    );
  }
```

It then makes it easier to figure out why the context menu is not working as an explicit error is thrown instead of an empty void function firing in silence behind the scenes.